components/cd-date/cd-date.html.twig

```diff
@@ -1,15 +1,11 @@
 {{ attach_library('common_design/cd-date') }}
 
-<div class="cd-date cd-date-wrapper">
-  <div class="cd-date-month-wrapper">
-    <span class="cd-date-month">Jul</span>
+<div class="cd-date">
```

Comment:
Could the outer tag be `<time>` with `datetime` attribute to supply machine-readable output?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

The suggestion assumes we have a machine-readable date when using this template to output data. If we don't then we actualy shouldn't use `<time>`:

>  If the element does **not** have a `datetime` attribute, **it must not have any element descendants**, and the datetime value is the element’s child text content.

Reply:
I'm 💯 in favor of that. But, as often, Drupal is in the way and will strip out the `datetime` attribute... I've been bitten by this when working on ODSG and had to patch core: https://github.com/UN-OCHA/odsg8-site/blob/develop/patches/core--drupal--xss-prevent-protocol-stripping-on-datetime-attribute.patch. The list of accepted attributes is hardcoded so no way to deal with that in a `hook`.

Reply:
I guess no change here then. Forcing core patches alongside CD is not cool IMO.